modules/Sources/Dependencies/SupportDataGenerator/SupportDataGenerator.swift

```diff
@@ -43,6 +44,31 @@ public enum SupportDataGenerator {
 
         return SupportData(toAddress: Constants.email, subject: Constants.subject, message: message)
     }
+    
+    public static func generatePartialProposalError(txIds: [String]) -> SupportData {
+        let data = SupportDataGenerator.generate()
+        
+        let idsString = txIds.map { id in
+            "- ID: \(id)"
+        }
+        .joined(separator: "\n")
+
+        let message =
+        """
+        Hi Zashi Team,
+
+        While sending a transaction, I encountered an error state and I'm reaching out to get help with recovering my funds.
```

Comment:
As noted in the conversation on the PR, I disagree that zcash_client_backend should implement ZIP 320 in a way that leads to risk of funds loss or a user ever needing to do something special to recover funds. It is not necessary.